### ISSUE TITLE
modified fast neural net losses

### ISSUE THREAD

**Issue by mmkqasim** [jcjohnson/fast-neural-style#124]
Hi, 

I have pruned fast neural net based on the magnitude of the weights. Now, I want to evaluate the prediction loss. I have gone through slow_neural_style.lua and train.lua to figure out how the losses are computed. I have tried to do the same but with no success. Is there a simple script that I could write to evaluate the perceptual loss or total loss, without training. I don't want to train. All I want is to evaluate the losses after pruning/modification. 

Greatly appreciate your response. 

Thanks!

**Comment by htoyryla** [comment]
If I am not mistaken, fast-neural-style calculates losses only during training. When the trained model is used to process an image, loss calculation is not needed and not used. Furthermore, during training losses are evaluated using a VGG network, which is not used by fast-neural-style after training. So if you want to evaluate losses for a fast-neural-style transfer, you would have to implement a script that takes the content, style and result images and uses a vgg network to capture targets and calculate the loss. I am not sure how useful this is.
